src/components/UserProfile.jsx

```diff
@@ -109,7 +109,11 @@ export const UserDetails = ({
             placeholder='Your name'
           >
             <h2 className='user-profile-name'>
-              {user.Name ? user.Name : user.username}
+              {user.Name
+                ? user.Name === 'null'
```

Comment:
why do we have `null` as string? @SundeepChand do you have any idea?

Reply:
also, looks like this ` == 'null'` comparison is being used in multiple places, let's make a function for it and use it everywhere